relay-event-normalization/src/timestamp.rs

```diff
@@ -0,0 +1,145 @@
+use relay_event_schema::processor::{
+    ProcessingAction, ProcessingResult, ProcessingState, Processor,
+};
+use relay_event_schema::protocol::Event;
+use relay_protocol::Meta;
+
+/// Ensures an event's timestamps are not stale.
+///
+/// Stale timestamps are those that happened before January 1, 1970 UTC. The
+/// processor validates the start and end timestamps of an event and returns an
+/// error if any of these is stale.
```

Comment:
```suggestion
/// error if any of these are stale.
```

(i feel so petty doing this)